packages/configs/webextension/package.json

```diff
@@ -23,5 +23,10 @@
   },
   "peerDependencies": {
     "@parcel/core": "^2.5.0"
+  },
+  "@parcel/bundler-default": {
+    "minBundles": 10000000,
+    "minBundleSize": 3000,
+    "maxParallelRequests": 20
```

Comment:
We should address this in a separate PR, for now this should probably be removed.

Reply:
Indeed.  That's a workaround for https://github.com/parcel-bundler/parcel/issues/8071 that ended up in this one.  Will remove.